jenkins-scripts/lib/project-default-devel-homebrew-amd64.bash

```diff
@@ -110,9 +110,9 @@ export DISPLAY=$(ps ax \
   | sed -e 's@.*Xquartz @@' -e 's@ .*@@'
 )
 
-# set CMAKE_PREFIX_PATH if we are using qt5 (aka qt)
-if brew ruby -e "exit ! '${PROJECT_FORMULA}'.f.recursive_dependencies.map(&:name).keep_if { |d| d == 'qt' }.empty?"; then
-  export CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/usr/local/opt/qt
+# set CMAKE_PREFIX_PATH if we are using qt@5
```

Comment:
changes looks good to me. Thinking on making the maintainership effort lower, do you see any hurt in making the export clause unconditionally in all our build?

Reply:
I suppose we could check instead if the folder exists, and in that case add it to `CMAKE_PREFIX_PATH`, that should be functionally equivalent but a bit simpler

can we do that in a follow-up PR?

Reply:
I'll do it in a follow-up PR